common/views.py

```diff
@@ -244,7 +219,9 @@ def post(self, request, *args, **kwargs):
                     logger.info("Updated Department: " + str(self.get_user_profile_info(request).department))
 
             except Exception as e:
-                return HttpResponse("Error updating records: " + str(e))
+                # For the superuser it is expected to not to be able to update records
+                # as it does not have a user profile object.
```

Comment:
That's what I'm talking about as well, may be we should actually generate this data for the super user as well. Let's discuss this with the team.